docs/clustering/multi-node-setup.rst

```diff
@@ -27,152 +26,129 @@ method of your choice.
     (which runs on every node) and checking the `cluster browser`_.
 
-    If you find yourself with multiple independent clusters and instead want to
+    If you end up with multiple independent clusters and instead want to
     form a single cluster, follow these steps:
 
       1. `Back up your data`_
       2. Shut down all the nodes
       3. Completely wipe each node by deleting the contents of the ``data``
          directory under `CRATE_HOME`_
-      4. Follow the instructions in the next section (:ref:`manual bootstrapping
+      4. Follow the instructions in :ref:`manual bootstrapping
          <manual-bootstrapping>`)
       5. Restart all the nodes and verify that they have formed a single cluster
       6. Restore your data
 
 
 .. _auto-bootstrapping-tarball:
 
-Tarball install
-~~~~~~~~~~~~~~~
+Tarball installation
+^^^^^^^^^^^^^^^^^^^^
 
-If you're installing CrateDB using the `tarball method`_, you can start a
-single host three-node cluster with auto-bootstrapping by following these
+If you are installing CrateDB using the `tarball method`_, you can start a
+single-host three-node cluster with auto-bootstrapping by following these
 instructions.
 
-Unpack the tarball:
-
-.. code-block:: console
-
-    sh$ tar -xzf crate-*.tar.gz
+1. Unpack the tarball:
 
-Copy the expanded directory three times, one for each node:
-
-.. code-block:: console
-
-    sh$ cp -R crate-*/ node-01
-    sh$ cp -R crate-*/ node-02
-    sh$ cp -R crate-*/ node-03
-
-.. TIP::
-
-    Each directory will function as `CRATE_HOME`_ for that node
-
-Because you want to run a multi-node cluster, you should configure the metadata
-gateway so that CrateDB knows how to recover its state safely. Ideally, for a
-three-node cluster, set `gateway-expected-nodes`_ to ``3`` and set
-`gateway-recover-after-nodes`_ to ``3``.
-
-.. NOTE::
+   .. code-block:: console
 
-    Configuring the metadata gateway is a safeguarding mechanism that is useful
-    for production clusters. It is not strictly necessary when running in
-    development. However, the `Admin UI`_ will issue warnings if you have not
-    configured the metadata gateway.
+       sh$ tar -xzf crate-*.tar.gz
 
-You can specify both settings in your `configuration`_ file, like so:
+2. Copy the unpacked directory into a new directory, three times, one for each
+   node. For example:
 
-.. code-block:: yaml
+   .. code-block:: console
 
-    gateway:
-      recover_after_nodes: 3
-      expected_nodes: 3
+       sh$ cp -R crate-*/ node-01
+       sh$ cp -R crate-*/ node-02
+       sh$ cp -R crate-*/ node-03
 
-Alternatively, you can configure this setting at startup with command-line
-options:
+   .. TIP::
 
-.. code-block:: console
+      Each directory will function as `CRATE_HOME`_ for that node
 
-    sh$ bin/crate \
-        -Cgateway.expected_nodes=3 \
-        -Cgateway.recover_after_nodes=3
+3. You should configure the :ref:`metadata gateway <metadata-gateway>` so that
```

Comment:
would it make sense to configure that before copying it? also i would be specific on where to do that.

Reply:
Okay, I didn't actually try it out myself and was just rewording what was there.  But maybe I can try it out and discover for myself the flow. 

In terms of specifying where to do that, I was hoping that my linking to the example in the Metadata gateway section on the bottom would be sufficient.  Wdyt?

`The :ref:`metadata gateway <metadata-gateway>` section includes examples.`